packages/dom-utilities/tsconfig.json

```
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "dist",
+    "target": "es5",
+    "module": "commonjs",
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "moduleResolution": "node",
+    "preserveConstEnums": true,
+    "lib": ["es5", "dom"],
+    "typeRoots": ["../../node_modules/@types", "../../typings"],
+    "types": ["jest", "webpack-env", "custom-global"],
+    "paths": {
+      "@fluentui/dom-utilities/lib/*": ["./src/*"],
+      "@fluentui/dom-utilities": ["./src"]
+    }
```

You can probably simplify this as follows:
```suggestion
    "lib": ["es5", "dom"]
```

---

Sounds like someone should update the `create-package` tool, then. Or provide a base `tsconfig.json` for which we can use `extends`.

---

Not all the suggestions here apply universally, and some things (`paths` and `baseUrl`, maybe others) will always need to be specified in each package separately even if we have config extension. (Basically the reason we don't have config extension is that no one has had/made time to set it up.)

In the create-package template, we include the paths by default because they're needed in projects that have examples (which use the full package names for importing). This project will never have examples, so it doesn't need the paths. It would potentially be possible to add a question to create-package about examples to control whether this stuff is added.

For the types and typeRoots, I realize now that removing `jest` types was incorrect (not sure what I was thinking there, sorry). `webpack-env` and `custom-global` probably will not be strictly needed, but it doesn't hurt having them there, so the only reason for removing them was to make the config a little simpler.